src/code42cli/cmds/devices.py

```diff
@@ -208,8 +208,8 @@ def _get_device_info(sdk, device_guid):
     required=False,
     type=str,
     default=None,
-    help="Limit devices to only the ones in the org you specify. "
-    "Note that child orgs will be included.",
+    help="Limit devices to only those in the organization you specify. "
```

Comment:
Extra space after the period.

Reply:
This is a multiline string.  The space at the end is because it's two sentences